plugin/trino-hive/src/main/java/io/trino/plugin/hive/metastore/glue/PartitionName.java

```diff
@@ -17,9 +17,9 @@
 
 import static java.util.Objects.requireNonNull;
 
-record PartitionName(List<String> partitionValues)
+public record PartitionName(List<String> partitionValues)
```

Comment:
I have been thinking we should add this interface to our general Hive model, and update usages.  It just make is so much easier to understand what is going on.